UnitTestFrameworkPkg/Library/GoogleTestBadExamples/Googletest/GoogleTestBadExamplesUT.inf

```diff
@@ -0,0 +1,32 @@
+## @file
+# Host OS based Application that Unit Tests GoogleTestBadExamplesSrc
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION     = 0x00010005
+  BASE_NAME       = GoogleTestBadExamplesUT
+  FILE_GUID       = 445944BA-983D-4F0B-95E6-5AC5D521B66D
+  MODULE_TYPE     = HOST_APPLICATION
+  VERSION_STRING  = 1.0
+
+[Sources]
+  GoogleTestBadExamplesUT.cpp
+  ../GoogleTestBadExamplesSrc.c
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  PolicyServicePkg/PolicyServicePkg.dec

```

Comment:
Is there a plan to upstream PolicyServicePkg?

One of the problems with keeping it in mu_basecore long term is that it extends its reach into other packages as a dependency (like this) which increases the number of Mu overrides that can't be reconciled until the package is upstreamed or the code is refactored to eliminate the PolicyServicePkg dependency.

Reply:
Agree

Reply:
The usage of PolicyService is confusing matters. Removed reference to that. The real focus is on mocking the locate protocol here.